util/pom.xml

```diff
@@ -102,5 +102,16 @@
         <filtering>true</filtering>
       </resource>
     </resources>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration combine.children="append">
+          <argLine combine.children="append">
```

Comment:
Let's add a comment that specifies this is for the SocketUtilTest and that you can look there to see why it's necessary.